lib/commands/general.js

```diff
@@ -9,6 +9,10 @@ let extensions = {},
     commands = {},
     helpers = {};
 
+commands.active = async function () {
+  return await this.uiautomator2.jwproxy.command('/element/active', 'POST', {});
```

Comment:
the request method must be GET according to the W3C protocol: https://www.w3.org/TR/webdriver/#dfn-get-active-element

Reply:
hmm there is call for active element in `appium-xcuitest-driver` that is using `POST` method I wanted to get it consistent with that method https://github.com/appium/appium-xcuitest-driver/blob/70adbf60bbf66deb86388e602b53eae902a3e04f/lib/driver.js#L100

Reply:
Per the comment on https://github.com/appium/appium-uiautomator2-server/pull/424 we'll submit a PR to deprecate the `POST` of `/active` if it still exists on _XCUITest_ and add the `GET` variant.